index.bs

```diff
@@ -4777,6 +4781,92 @@ which should make use of the existing browser permissions framework for the Geol
 
 The privacy principles in [[!FIDO-Privacy-Principles]] also apply to this specification.
 
+
+## De-anonymization prevention measures ## {#sctn-privacy-attacks}
+
+[INFORMATIVE]
+
+Many aspects of the design of the [=Web Authentication API=] are motivated by privacy concerns. The main concern considered in
+this specification is the protection of the user's personal identity, i.e., the identification of a [=natural person=] or a
```

Comment:
see https://github.com/w3c/webauthn/pull/899#discussion_r195522026 -- human being is fine, silly me. sorry/thx.